src/Razor/Microsoft.NET.Sdk.Razor/test/IntegrationTests/BuildIntegrationTest11.cs

```diff
@@ -1,58 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I had some trouble with building the 1.1 projects so I'm leaving it out in this phase of the migration. I will revisit it after I stabilize the rest of migration.

Reply:
Please file an issue to track this work.

Reply:
I might address it by the time the entire PR is reviewed. I'll see where I end up.

Reply:
I got it, tests are added back.